quantum/config_common.h

```diff
@@ -17,8 +17,10 @@
 #pragma once
 
 /* diode directions */
-#define COL2ROW 0
-#define ROW2COL 1
+#define COL2ROW 1
```

Comment:
Any particular reasoning for the renumbering of these two defines?

Reply:
Without that, something like this will compile. This as just an example.
#define DIODE_DIRECTION RAW2CAL
And, Instead of warned or reported with an error, it will be treated as COL2ROW.